src/frontend/Sider/Behandling/Stønadsvilkår/DagligReise/OffentligTransportSeksjon.tsx

```diff
@@ -90,43 +47,73 @@ export const OffentligTransportSeksjon = ({
                     <TextField
                         label={'Pris enkeltbillett'}
                         size="small"
-                        value={harTallverdi(enkeltbillett) ? enkeltbillett : ''}
+                        value={
+                            harTallverdi(offentligTransport?.prisEnkelbillett)
+                                ? offentligTransport.prisEnkelbillett
+                                : ''
+                        }
                         onChange={(e) => {
-                            oppdaterEnkeltbilett(tilHeltall(fjernSpaces(e.target.value)));
+                            oppdaterFelt(
+                                'prisEnkelbillett',
+                                tilHeltall(fjernSpaces(e.target.value))
+                            );
                         }}
                     />
                 </FeilmeldingMaksBredde>
                 <FeilmeldingMaksBredde $maxWidth={180}>
                     <TextField
                         label={'Pris 7-dagersbillett'}
                         size="small"
-                        value={harTallverdi(syvdagersbilett) ? syvdagersbilett : ''}
+                        value={
+                            harTallverdi(offentligTransport?.prisSyvdagersbillett)
+                                ? offentligTransport?.prisSyvdagersbillett
+                                : ''
+                        }
                         onChange={(e) => {
-                            oppdaterSyvdagersBilett(tilHeltall(fjernSpaces(e.target.value)));
+                            oppdaterFelt(
+                                'prisSyvdagersbillett',
+                                tilHeltall(fjernSpaces(e.target.value))
+                            );
                         }}
                     />
                 </FeilmeldingMaksBredde>
                 <FeilmeldingMaksBredde $maxWidth={180}>
                     <TextField
                         label={'Pris 30-dagersbillett'}
                         size="small"
-                        value={harTallverdi(trettidagersbillett) ? trettidagersbillett : ''}
+                        value={
+                            harTallverdi(offentligTransport?.prisTrettidagersbillett)
+                                ? offentligTransport?.prisTrettidagersbillett
+                                : ''
+                        }
                         onChange={(e) => {
-                            oppdaterTrettidagersbilett(tilHeltall(fjernSpaces(e.target.value)));
+                            oppdaterFelt(
+                                'prisTrettidagersbillett',
+                                tilHeltall(fjernSpaces(e.target.value))
+                            );
                         }}
                     />
                 </FeilmeldingMaksBredde>
             </HStack>
+            <ErrorMessage size="small">{feilmeldinger.billettpriser}</ErrorMessage>
```

Comment:
Kunne vurdert her at alle de tre feltene for billettpris ble rød i tillegg til at det ble vist en feilmelding under evt.